packages/demo/src/helpers/launchEditor.ts

```diff
@@ -76,6 +82,50 @@ export function launchEditor(currentSmartFilter: SmartFilter, engine: ThinEngine
                     currentSmartFilter.name + ".json"
                 );
             },
+            loadSmartFilter: async (file: File) => {
+                return smartFilterLoader.loadFromFile(file, false); // TODO: update optimize
+            },
+            customSave: async () => {
```

Comment:
The name customSave doesn't tell me what kind of save we're doing - how about naming this saveToSnippetServer and renaming saveSmartFilter above to downloadSmartFilter?

I haven't looked yet, but I'm curious about how these two are invoked differently in the editor now.

Reply:
(See propertyTabComponent)